### ISSUE TITLE
consistent mouse lag when mumble is open

### ISSUE THREAD

**Issue by efiltsohg** [mumble-voip/mumble#3534]
Whenever mumble is open, my mouse cursor moves at roughly half of normal speed & about half of mouse inputs are lost (e.g. have to click 2 or 3 times to make a click register, in any window not just in mumble window)

I have tried 4 different releases, same problem.  No controller connected and I disabled xboxinput via registry.  Windows 10 Home 1809 build 17763.55, fresh install, happens regardless of other software open.  No mouse software installed.

The instant I quit mumble my mouse goes back to normal.

**Comment by efiltsohg** [comment]
On further testing, this only happens when my driverless USB sound card (which is required for my microphone to work) is connected

**Comment by Avamander** [comment]
Does this happen with just a voice recording software?

**Comment by efiltsohg** [comment]
Seems to be mumble only

**Comment by Scefing** [comment]
I seem to have this problem too, in 1.30.0-rc1. `Console.txt` log has input like this at the lag times:
```
<W>2019-04-12 21:01:08.978 GlobalShortcutWin: rejected blacklisted device HID-compliant consumer control device (GUID: {bb80a440-b81a-11e7-8001-444553540000}, PGUID: {10aa262a-0000-0000-0000-504944564944}, VID: 0x262a, PID: 0x10aa, TYPE: 0x00010011)
<W>2019-04-12 21:01:19.049 GlobalShortcutWin: rejected blacklisted device HID-compliant consumer control device (GUID: {bb80a440-b81a-11e7-8001-444553540000}, PGUID: {10aa262a-0000-0000-0000-504944564944}, VID: 0x262a, PID: 0x10aa, TYPE: 0x00010011)
<W>2019-04-12 21:06:33.954 GlobalShortcutWin: rejected blacklisted device HID-compliant consumer control device (GUID: {bb80a440-b81a-11e7-8001-444553540000}, PGUID: {10aa262a-0000-0000-0000-504944564944}, VID: 0x262a, PID: 0x10aa, TYPE: 0x00010011)
<W>2019-04-12 21:06:44.015 GlobalShortcutWin: rejected blacklisted device HID-compliant consumer control device (GUID: {bb80a440-b81a-11e7-8001-444553540000}, PGUID: {10aa262a-0000-0000-0000-504944564944}, VID: 0x262a, PID: 0x10aa, TYPE: 0x00010011)
```
I also have a driverless USB sound card.

**Comment by jameskimmel** [comment]
some problem here with Antlion Audio USB Sound Card. Makes Mumble unfortunately unusable for me. Is there no fix for this problem?

**Comment by Krzmbrzl** [comment]
This is probably going to be solved as soon as #4039 has been implemented. This is currently worked on, but I can't give you an ETA yet.

**Comment by davidebeatrici** [comment]
Closing in favor of #4039.
